Podcast/Player.swift

```diff
@@ -41,10 +35,13 @@ class Player: NSObject {
         didSet {
             autoplay = true
             currentItemPrepared = false
+            scrubbing = false
+            delegate?.updateUI()
         }
     }
     private var autoplay: Bool
     private var currentItemPrepared: Bool
```

Comment:
I chose to make it private since there is a lot of complicated logic with KVO/etc. for clearing out any previously playing items and asynchronously loading a new Episode. So, I'd prefer that the Player handle all this logic when `playEpisode(Episode)` is called. I agree the `didSet` logic is a little confusing so I'll put it in a `reset()` function.